### PR TITLE
Remove unnecessary step csv to tabular

### PR DESCRIPTION
**Pull request by yvanlebras**

### CODE REVIEW

File: topics/ecology/tutorials/regionalGAM/tutorial.md [hunk 1]
```diff
@@ -318,8 +309,9 @@ This shows the abundance of *Pyronia tithonus*, over the weeks. We can see there
 >    - *"Plot title"*: add a meaningful title (e.g. `Pyronia tithonus phenology stacked visualization`)
 >    - *"Label for x axis"*: add a meaningful label (e.g. `Day number`)
 >    - *"Label for y axis"*: add a meaningful label <!-- TODO (e.g. ????? Ask Benjamin or Reto to know more)-->
->    - *"Type of plot"*: `Line only`
+>    - *"Type of plot"*: `Line only` (e.g. `nm value`)
```
Comment (shiltemann) [inline topics/ecology/tutorials/regionalGAM/tutorial.md]: did you mean to put this on the line above (y-axis) or?

Comment (yvanlebras) [reply topics/ecology/tutorials/regionalGAM/tutorial.md]: oh no...  A mistake sorry!

Comment (yvanlebras) [reply topics/ecology/tutorials/regionalGAM/tutorial.md]: Copy / paste error, sorry. Corrected now!

**Comment by yvanlebras** [comment]
Thank you very much

**Pull request merged**